src/main/java/ch/njol/skript/bukkitutil/BukkitUnsafe.java

```diff
@@ -70,7 +71,10 @@ public class BukkitUnsafe {
 
 	@Nullable
 	public static Material getMaterialFromMinecraftId(String id) {
-		return Material.matchMaterial(id);
+		return Material.matchMaterial(id.toLowerCase().startsWith(NamespacedKey.MINECRAFT + ":")
+										  ? id
+										  : id.replace(":", "_")  //For Hybrid Server
+		);
```

Comment:
we probably shouldn't be handling non-minecraft ids in the method that says it handles minecraft ids

Reply:
`getMaterialFromNamespacedId`  (omg what am I saying)

Reply:
Sure, have that call the existing minecraft one if it starts with minecraft, otherwise do your new logic

Reply:
All right let's try to rename it (will it be okay?

Reply:
Renaming would be a breaking change, so it'd be better to add a new method and route things through that instead.
Though maybe i'm making a mountain out of a molehill.

Reply:
> Renaming would be a breaking change, so it'd be better to add a new method and route things through that instead. Though maybe i'm making a mountain out of a molehill.

Maybe I can make a method named `getMaterialFromMinecraftId()` and call the `getMaterialFromNamespacedId()`, and marked as it is `@Deprecated` ?

Reply:
Sure, that works!